src/index.html

```diff
@@ -5,8 +5,412 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap"
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#" class="header__logo-link">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-list-item-1">
+            <a
+              href="#"
+              class="header__nav-list-item-1-link is-active"
+              data-qa="is-active"
+            >
+              apple
+            </a>
```

Comment:
make your class names simpler

```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a
              href="#"
              class="nav__link is-active"
              data-qa="is-active"
            >
              apple
            </a>
```